runtime/core/portable_type/tensor_impl.cpp

```diff
@@ -11,6 +11,8 @@
 #include <algorithm>
```

Comment:
not related to this diff per se, but makes me curious why we need <algorithm> in this file too.

Reply:
[std::equal](https://en.cppreference.com/w/cpp/algorithm/equal)